user/common-tasks/software-update-domu.md

```diff
@@ -198,6 +198,70 @@ Example policy file in R4.0 (with Whonix installed, but not set as default Updat
 @anyvm @anyvm deny
 ```
 
+# Installing Snap Packages
+
+Snap packages do not use the normal update channels for Debian and Fedora (apt and dnf) and are often installed as the user rather than as root. To support these in an AppVM you need to take the following steps:
+
+1. In the **TemplateVM** you must install snapd and qubes-snapd-helper:
+Open a terminal in TemplateVM and run:
+```shell_session
+[user@fedora-30-snap-demo ~]$ sudo dnf install snapd qubes-snapd-helper
+Last metadata expiration check: 0:55:39 ago on Thu Nov 14 09:26:47 2019.
+Dependencies resolved.
+========================================================================================================
+ Package                       Arch    Version                             Repository              Size
+========================================================================================================
+Installing:
+ snapd                         x86_64  2.42.1-1.fc30                       updates                 17 M
+ qubes-snapd-helper            noarch  1.0.1-1.fc30                        qubes-vm-r4.0-current   10 k
+Installing dependencies:
+[...]
+
+Transaction Summary
+========================================================================================================
+Install  20 Packages
+
+Total download size: 37 M
+Installed size: 121 M
+Is this ok [y/N]: y
+
+Downloading Packages:
+[..]
+Failed to resolve booleanif statement at /var/lib/selinux/targeted/tmp/modules/200/snappy/cil:1174
+/usr/sbin/semodule:  Failed!
+[...]
+Last metadata expiration check: 0:57:08 ago on Thu Nov 14 09:26:47 2019.
+Notifying dom0 about installed applications
+
+Installed:
+  snapd-2.42.1-1.fc30.x86_64                                              qubes-snapd-helper-1.0.1-1.fc30.noarch                           
+[...]                          
+Complete!
+```
+You may see a message "Failed to resolve booleanif statement at /var/lib/selinux/targeted/tmp/modules/200/snappy/cil:1174
+/usr/sbin/semodule:  Failed!". This is expected and you can safely continue.
+
+Shutdown the TemplateVM:
+```shell_session
+[user@fedora-30-snap-demo ~]$ sudo shutdown -h now
+```
+
+2. Now open the **AppVM** in which you would like to install the Snap application and run a terminal:
+```shell_session
+[user@snap-demo-AppVM ~]$ snap install <package>
+```
+when the install is complete shutdown the AppVM:
+```shell_session
+[user@snap-demo-AppVM ~]$ sudo shutdown -h now
```

Comment:
Shutting down AppVM isn't necessary step. Especially, the next step (refreshing application list) will start it again.